odoo/addons/base/tests/config/16.0.conf

```diff
@@ -61,7 +61,7 @@ upgrade_path =
 websocket_keep_alive_timeout = 3600
 websocket_rate_limit_burst = 10
 websocket_rate_limit_delay = 0.2
-without_demo = False
+with_demo = False
```

Comment:
you never change this file
this file exist to make sure that a configuration generated for a previous version still work with the latest version
it is acceptable that new warnings are emitted